### ISSUE TITLE
Daily/weekly/monthly mortality overall in Italy for 2018/20119/2020 (not just covid-19 cases)

### ISSUE THREAD

**Issue by janrth** [pcm-dpc/COVID-19#413]
I would like to isolate the effect that covid-19 had on the mortality. I am planning to apply a method that reduces some bias due to unclear behavior in case of death. It is reported that if somebody dies, who was positive for covid-19 he or she is counted in the covid-19 statistic. But the actual reason of death might be different.
This is why I am planning to forecast the expected number of death for Italy (probably on a monthly basis) using some time-series forecast method. Then I can compare my prediction to the actual number of overall mortality. The difference between these two, would be my covid-19 effect. 
I am trying to get some time-series data, but I have only found some data up to 2017 for Italy. It would be great if you could help me out with more data here!

**Comment by Tantormai** [comment]
The latest data I found on ISTAT site (National Institute of Statistics) are those of 2018.
It seems that today that site is busy:
http://dati.istat.it/Index.aspx?DataSetCode=DCIS_MORTALITA1#

**Comment by janrth** [comment]
I also found that data from 2018. Problem is I would need time-series data, plus actual mortality data from 2020. 
I also found this here: http://data.un.org/Data.aspx?d=POP&f=tableCode%3A65
That is actually nice. There is mortality from other countries like Germany until most of 2019. I can probably predict IT data based on that. 
It would also be possible to create monthly data from aggregated 2019 data (by applying a general monthly mortality share - I think it is rather stable on country level). 
But it seems that it is hard to find most recent data.
